samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -119,14 +119,18 @@ class MyVirtualList extends Component {
 					direction="vertical"
 					itemRenderer={this.renderItem}
 					itemSize={{minWidth: ri.scale(570), minHeight: ri.scale(156)}}
-					key={select('scrollMode', prop.scrollModeOption, Config)}
-					scrollMode={select('scrollMode', prop.scrollModeOption, Config)}
+					key={this.props.args['scrollMode']}
+					scrollMode={this.props.args['scrollMode']}
```

Comment:
You don't need to refer `this` since we already have `props` in render function.
```suggestion
					key={props.args['scrollMode']}
					scrollMode={props.args['scrollMode']}
```

Reply:
OK